index.html

```diff
@@ -41,13 +70,13 @@ <h2 class="introduction">I'm a software developer</h2>
    </div>
    
   </div>
-
   
 </section>
 
 <section class="works">
```

Comment:
- [ ] I kindly suggest that you follow the Figma design for your project because it's a part of the project requirements. Also please ensure you have 2 rows with 3 projects each as shown in the Template Design

| Template Design  | Your Design |
| ------------- | ------------- |
| ![Screenshot from 2022-05-07 23-42-05](https://user-images.githubusercontent.com/52934128/167274156-73846026-c2bb-4b9d-b62f-e94b5384a674.png)  | ![Screenshot from 2022-05-07 23-37-24](https://user-images.githubusercontent.com/52934128/167274162-56cfd398-da94-42e7-8ced-7b33785a0707.png)  |